projects/js-packages/charts/src/charts/segmented-bar/segmented-bar.module.scss

```diff
@@ -0,0 +1,105 @@
+.segmentedBar {
+	// CSS custom properties for theming
+	--segmented-bar-empty-bg: #e5e7eb;
+	--segmented-bar-label-color: #6b7280;
+	--segmented-bar-label-font-size: 12px;
+	--segmented-bar-tooltip-bg: #1f2937;
+	--segmented-bar-tooltip-color: #fff;
+	--segmented-bar-tooltip-font-size: 12px;
+	--segmented-bar-segment-hover-opacity: 0.8;
+	--segmented-bar-marker-width: 2px;
```

Comment:
This is inconsistent with how we theme other charts. What's the reason for diverging from using theme settings? If we are to do this I'd expect to use wpds tokens

Reply:
IMO it's extra effort and cognitive load for the consumer to add styles in 3 places; marker color in the theme, segment colors in the props, all the other colors here...